src/reducers/sort.js

```diff
@@ -0,0 +1,11 @@
+import * as constants from '../constants.json';
+
+const actions = {
+  [constants.SORT_BY_RATE]: (state, action) => action.payload,
```

Comment:
As we have identical functions here, I would suggest to create such function outside of object